go.mod

```diff
@@ -3,75 +3,89 @@ module github.com/indykite/terraform-provider-indykite
 go 1.19
 
 require (
-	github.com/daixiang0/gci v0.9.1
+	github.com/daixiang0/gci v0.10.1
 	github.com/golang/mock v1.6.0
-	github.com/golangci/golangci-lint v1.51.2
+	github.com/golangci/golangci-lint v1.52.2
 	github.com/hashicorp/go-cty v1.4.1-0.20200723130312-85980079f637
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
 	github.com/hashicorp/terraform-plugin-go v0.14.3
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.25.0
-	github.com/indykite/jarvis-sdk-go v0.5.0
-	github.com/onsi/ginkgo/v2 v2.8.4
-	github.com/onsi/gomega v1.27.2
-	google.golang.org/grpc v1.53.0
-	google.golang.org/protobuf v1.28.2-0.20230222093303-bc1253ad3743
+	github.com/indykite/jarvis-sdk-go v0.8.0
```

Comment:
FYI: Yesterday I create v `0.9.0`
Shouldn't have any changes related to this thou

Reply:
But good to be always up to date. At least, you will not get warning from Renovate again